2023-06-30 01:09:44,169 TADA INFO starting test `ldmsd_auth_test`
2023-06-30 01:09:44,170 TADA INFO   test-id: d979f1a09ec7ed44e7c6622b186b368aa63ccd5a3b47cf72c5f3025c29253e6d
2023-06-30 01:09:44,170 TADA INFO   test-suite: LDMSD
2023-06-30 01:09:44,170 TADA INFO   test-name: ldmsd_auth_test
2023-06-30 01:09:44,170 TADA INFO   test-user: narate
2023-06-30 01:09:44,170 TADA INFO   commit-id: 26f8c63d31947afe7b20b74b1c158c9fdd735a13
2023-06-30 01:09:44,170 __main__ INFO -- Get or create the cluster --
2023-06-30 01:10:02,317 __main__ INFO -- Start daemons --
2023-06-30 01:10:48,771 __main__ INFO ... wait a bit to make sure ldmsd's are up
2023-06-30 01:10:53,901 TADA INFO assertion 1, root@agg-2(dom3) ldms_ls to agg-2:10000: see all sets, passed
2023-06-30 01:10:54,021 TADA INFO assertion 2, user@agg-2(dom3) ldms_ls to agg-2:10000: see only meminfo, passed
2023-06-30 01:10:54,158 TADA INFO assertion 3, root@headnode(dom4) ldms_ls to agg-2:10001: see all sets, passed
2023-06-30 01:10:54,279 TADA INFO assertion 4, user@headnode(dom4) ldms_ls to agg-2:10001: see only meminfo, passed
2023-06-30 01:10:54,399 TADA INFO assertion 5, root@headnode(dom4) ldms_ls to agg-11:10000: unexpected result: ['Fri Jun 30 01:10:54 2023:    ERROR: auth_munge: munge_decode() failed. Invalid credential', 'Connection failed/rejected.'], failed
2023-06-30 01:10:54,399 TADA INFO test ldmsd_auth_test ended
